super-csv/src/main/java/org/supercsv/io/CsvBeanReader.java

```diff
@@ -17,7 +17,7 @@
 
 import java.io.IOException;
 import java.io.Reader;
-import java.lang.reflect.Method;
+import java.lang.reflect.*;
```

Comment:
Please, do not use stars in imports section. To avoid situation like this please use in your IDE this formatter: https://github.com/super-csv/super-csv/blob/master/supercsv_eclipse_formatter.xml